### PR TITLE
docs: improve the "Getting started" section

### PR DESCRIPTION
**Pull request by Wertzui123**

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ If V is already installed on a machine, it can be upgraded to its latest version
 by using the V's built-in self-updater.
 To do so, run the command `v up`.
 
-## Getting start
-In your terminal, you can:
-1) `v init` -> create a new project in the current folder
-2) `v new abc` -> create a new project in the new folder `abc`, by default a "hello world" project.
-3) `v new abcd web` -> create a new project in the new folder `abcd`, using the vweb template.
+## Getting started
+You can let V automatically set up the bare-bones structure of a project for you by using *one* of the following commands in a terminal:
```
Comment (JalonSolov) [inline doc/docs.md:39]:
```suggestion
You can let V automatically set up the bare-bones structure of a project for you by using any of the following commands in a terminal:
```

File: doc/docs.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ If V is already installed on a machine, it can be upgraded to its latest version
...
+* `v init` → creates a new project in the current folder
```
Comment (JalonSolov) [inline doc/docs.md:40]:
```suggestion
* `v init` → adds necessary files to the current folder to make it a V project
```

**Comment by spytheman** [comment]
Also wrap .md lines to a maximum of 100 characters per line.

File: doc/docs.md (hunk 2 above)
Comment (JalonSolov) [reply doc/docs.md]: Marked resolved, but no changes made?  Do you disagree with them?

Comment (Wertzui123) [reply doc/docs.md]: I did commit them in c345db154f0b6bd388d020ca4fb7d6cae934ecc7.

Comment (JalonSolov) [reply doc/docs.md]: Hmm... I didn't see that when I looked earlier.  Never mind, then. :-)

**Pull request merged**